### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by erwingelissen**
I would like to use this module, but my Django app runs on python3. I tried porting this module, but there are still some issues when running the tests. Any help?

### CODE REVIEW

**Comment by rudolphfroger** [comment]
Thanks for you pull request! I'd like to help out with full Python 3 compatibility.

File: django_auth_policy/authentication.py [hunk 1]
```diff
@@ -47,13 +47,13 @@ class AuthenticationBasicChecks(AuthenticationPolicy):
 
     def pre_auth_check(self, loginattempt, password):
         if not loginattempt.username:
-            logger.info(u'Authentication failure, address=%s, '
```
Comment (rudolphfroger) [inline django_auth_policy/authentication.py]: For every file where we remove the "u" prefix, we should add a "from __future__ import unicode_literals" statement, I assume. Could you change that?

File: django_auth_policy/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # encoding: utf-8
-from __future__ import unicode_literals
```
Comment (rudolphfroger) [inline django_auth_policy/migrations/0001_initial.py]: Probably we shouldn't remove these statements, see:
https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions

File: django_auth_policy/management/commands/unlock_all.py [hunk 3]
```diff
@@ -9,4 +9,4 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         c = LoginAttempt.objects.filter(lockout=True).update(lockout=False)
-        print u'Unlocked {0} login attempts.'.format(c)
```
Comment (rudolphfroger) [inline django_auth_policy/management/commands/unlock_all.py]: Please add:
from __future__ import print_function
to keep Python 2.7 compatibility

**Comment by erwingelissen** [comment]
Including the migrations causes "TypeError: Type str doesn't support the buffer API" and on top of that there are still 3 tests that fail.

**Comment by rudolphfroger** [comment]
I just decided that Python 3 support is no longer a priority for this project, see updated README file for details.

**Pull request closed without merging**